### PR TITLE
S5t5 reset course form

### PR DESCRIPTION
**Pull request by nesclark**
- resetting new fields to null (add in line below line 58 on addCourse.js)
- Review unit tests and update as necessary
- Ensure documentation is fully updated to include new functionality of this story

### CODE REVIEW

**Review by Steph-ski — APPROVED** [review]
Read me /addCourse looks good. Have tested in the browsers and reset is working.

File: README.md [hunk 1]
```diff
@@ -213,3 +213,17 @@ The score should be an integer 0-100.
   - If invalid data supplied
     - `{"success": false, "message": "Aptitude score not added - invalid data provided.", "data": []}`
   
+
+**/addCourse**
+
+POST
+- Adds a new course to the 'courses' table in the database.
```
Comment (HadertS) [inline README.md:220]: To be consistent with the rest of the readme I think the quotes around courses should be removed

File: README.md [hunk 2]
```diff
@@ -213,3 +213,17 @@ The score should be an integer 0-100.
...
+- Sends:
+  - `{"courseName":"example","startDate":"2023-04-12","endDate":"2024-04-12","trainer":"['1']","notes":"example","in_person":"0","remote":"1"}`
+  - If `remote` or `in_person` is `1` (true) then sends data regarding number of available spaces on the course e.g:
```
Comment (HadertS) [inline README.md:223]: I think eg should be removed for consistency

File: README.md [hunk 3]
```diff
@@ -213,3 +213,17 @@ The score should be an integer 0-100.
...
+  - `{"remote_spaces":"5","in_person_spaces":"2"}`
```
Comment (HadertS) [inline README.md:224]: Indent this

**Review by HadertS — CHANGES_REQUESTED** [review]
A couple of minor formatting changes on the readme to make it consistent with the overall style

**Review by HadertS — APPROVED** [review]

**Pull request merged**